object-browser-schema.js

```diff
@@ -0,0 +1,199 @@
+export default {
+  additionalProperties: false,
+  definitions: {
+    actionItemDisabled: {
+      description: 'Tells if action bar button or link is disabled or ties it to selection',
+      oneOf: [
+        {
+          type: 'boolean',
+          default: false,
+        }, {
+          type: 'object',
+          additionalProperties: false,
+          description: 'Shortcut to enable and disable button or link as selection changes',
+          properties: {
+            minSelected: {type: 'number'},
+            maxSelected: {type: 'number'}
+          }
+        }
+      ]
+    },
+    actionItem: {
+      type: 'object',
+      properties: {
+        id: {
```

Comment:
If we give an actual handler function then it's no longer JSON. We could use named actions. Still, I think a single handler with id switch is less error prone and easier to debug.